tripleo-heat-templates-diff/common/diff_common_8.0.7-21

```diff
@@ -59,3 +59,80 @@ diff -r -u -N openstack-tripleo-heat-templates-original/docker/services/neutron-
          - name: Neutron package update workaround
            yum: name=python-networking-odl state=latest
            when:
+diff -ruN openstack-tripleo-heat-templates-original/puppet/services/neutron-compute-plugin-nuage.yaml openstack-tripleo-heat-templates/puppet/services/neutron-compute-plugin-nuage.yaml
+--- openstack-tripleo-heat-templates-original/puppet/services/neutron-compute-plugin-nuage.yaml 2019-01-31 00:27:45.770000000 -0500
++++ openstack-tripleo-heat-templates/puppet/services/neutron-compute-plugin-nuage.yaml  2019-01-31 00:26:21.594000000 -0500
+@@ -75,6 +75,25 @@
+     description: Support for non-default MTU configured on each Compute node
+     type: string
+     default: ''
++  OvsHwOffload:
++    default: false
++    description: |
++      Enable OVS Hardware Offload. This feature supported from OVS 2.8.0
```

Comment:
Same as above. Good to leave out OVS version.

Reply:
Ok, here we can make that change and I can remove "This feature supported from OVS 2.8.0" from the description
 cc @sunny-verma